hokusai/staging.yml

```diff
@@ -83,19 +83,25 @@ spec:
                       - foreground
 
 ---
-apiVersion: autoscaling/v1
+apiVersion: autoscaling/v2beta2
 kind: HorizontalPodAutoscaler
 metadata:
   name: force-web
   namespace: default
 spec:
-  scaleTargetRef:
```

Comment:
This seems to be required according to the [spec](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2beta2/#HorizontalPodAutoscalerSpec)
> scaleTargetRef (CrossVersionObjectReference), required